src/encoded/schemas/treatment.json

```diff
@@ -94,7 +94,9 @@
             "enum": [
                 "activation",
                 "deactivation",
-                "stimulation"
+                "stimulation",
```

Comment:
Can these be added in alphabetical order?

Reply:
done